devbox.json

```diff
@@ -17,11 +18,12 @@
     "python@3.11.2",
     "shellcheck@0.9.0",
     "yamllint@1.28.0",
+    "zlib@1.2.13",
```

Comment:
Adding `zlib` to our dependencies fixed the following error when I ran `make full-ent FIDO2=dynamic`:

```
ld: file not found: /usr/lib/libz.1.dylib for architecture arm64
```